tests/appsec/appsec_ato_sdk.json

```
@@ -0,0 +1,90 @@
+{
+  "version": "2.1",
+  "metadata": {
+    "rules_version": "1.2.6"
+  },
+  "rules": [
+    {
+      "id": "001_trigger_on_usr_login",
+      "name": "Trigger on usr.login",
+      "tags": {
+        "type": "security_scanner",
+        "category": "attack_attempt"
+      },
+      "conditions": [
+        {
+          "parameters": {
+            "inputs": [
+              {
+                "address": "usr.login"
+              }
+            ],
+            "regex": "login_unsafe"
+          },
+          "operator": "match_regex"
```

regex isn't needed. same thing line 45

```suggestion
            "list": ["login_unsafe"]
          },
          "operator": "exact_match"
```